source/user_guide.rst

```diff
@@ -293,11 +304,11 @@ that, we will use mosquitto_pub from Mosquitto project.
     `mosquitto`_ - one containing tools to access it (i.e. mosquitto_pub and
     mosquitto_sub for publishing messages and subscribing to topics) and
     another one containing the MQTT broker. In this tutorial, only the tools
-    are going to be used. Please check if MQTT broker is not running before 
+    are going to be used. Please check if MQTT broker is not running before
     starting dojot (by running commands like ``ps aux | grep mosquitto``).
 
 
-The dojot compatible format for messages sent by devices is a simple key-value 
+The dojot compatible format for messages sent by devices is a simple key-value
```

Comment:
Is there such thing as a dojot compatible format? I remember that somewhere on other part of the document, it said that anything was allowed